### DISCUSSION TITLE
Wordpress Plugin "show on selected pages" not working

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#191] [! Welcome - readme first!]
The `on selected pages` feature in the plugin doesn't look like it's complete. all of the selectable pages listed have the same generic title and a hard-coded value attribute in the option.

Would be great to be able to show patches on different selected pages.

**Comment by steam0r** [comment]
you are right, this was leftover frontend code for a feature that never made it into the plugin, i removed it in the new version. we are currently not actively developing this plugin as we are focusing on cables development. this would be a dependency on the page templates your theme provides and thus has to be tested and developed further as wordpress changes.

for now your best bet is to use the shortcode to include your patch on different pages and maybe use a customfield to decide on which page-template to show. there is other plugins that do offer shortcode inclusion on selected pages only, so i'd offload this feature to them for now.

**Reply by andrejhronco** [reply]
If this feature is removed now, how can you select (download to WP) a patch to use via short code or inclusion in a template?
Is selecting a patch in step 1 of the integration setup enough to download the patch to `wp-content/plugins/cables-wpplugin/public/patches` and store it permanently?

I have my page templates setup now with a customfield for the patch ID like you suggested and it works fine. 
But would be nice to have some UI for this case (select and download a patch to WP to reference it by patch ID) and to be able to delete the custom integration. 

Also, what does `Custom Integration` do in `Where do you want your patch to show?`

Appreciate the work on this plugin, I find it quite handy so i'm hoping that dev on this can continue to some extent.

**Reply by steam0r** [reply]
hey, documentation on how to use shortcodes with a copy paste example is right on the patch page:

![Bildschirmfoto 2022-03-10 um 09 54 00](https://user-images.githubusercontent.com/151271/157626087-73f5c068-6f9e-4b33-9a2f-6cd502b9ba46.png)

select "custom integration" to have the patch not automatically appear in the header/background/footer, so basically the right choice when using shortcode (only). and yes, the patch will then be downloaded and permanentely stored in the wordpress plugins folder.

**Reply by andrejhronco** [reply]
Thanks for the info, i'll try out those approaches
